stable_plugins/classical_ml/scikit_ml/optics-clustering/templates/optics-clustering_template.html

```diff
@@ -0,0 +1,102 @@
+{% extends "simple_template.html" %}
+
+{% block script %}
+<script>
+    var algorithm_enum_val = document.getElementById("algorithm_enum");
+    var metric_enum_val = document.getElementById("metric_enum");
+    var method_enum_val = document.getElementById("method_enum");
+    var epsilon_val = document.getElementById("epsilon");
+    var xi_val = document.getElementById("xi");
+    var min_cluster_size_val = document.getElementById("min_cluster_size");
+    var leaf_size_val = document.getElementById("leaf_size");
+    var minkowski_p_val = document.getElementById("minkowski_p");
+
+
+    var epsilon_dis = epsilon_val.parentNode.parentNode;
+    var xi_dis = xi_val.parentNode.parentNode;
+    var predecessor_correction_dis = document.getElementById("predecessor_correction").parentNode.parentNode;
+    var min_cluster_size_dis = min_cluster_size_val.parentNode.parentNode;
+    var leaf_size_dis = leaf_size_val.parentNode.parentNode;
+    var minkowski_p_dis = minkowski_p_val.parentNode.parentNode;
+
+    function set_union(setA, setB) {
```

Comment:
I think this can be replaced by https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/union

Reply:
unfortunately not. Efficient set methods are still experimental and without browser support (see bottom of the linked page). Until these methods are implemented by most browsers, sets in javascript will stay mostly useless.

Reply:
Thus, I should not replace this function?

Reply:
exactly, do not replace this.